app/cdap/components/Connections/ImportConnectionBtn/index.tsx

```diff
@@ -38,6 +38,12 @@ export default function ImportConnectionBtn({ onCreate, className = null }) {
 
   const classes = useStyle();
 
+  function handleFileClear() {
```

Comment:
This allows onChange to fire for same file

Reply:
why not add this as a comment to the code?